packages/tpetra/core/test/Map/Map_Bug5378.cpp

```diff
@@ -120,10 +124,10 @@ TEUCHOS_UNIT_TEST( Map, Bug5378_GoodGIDsNoLIDs )
   //
   // Lookup of any valid global ID should identify with proc 0
   //
-  RCP<const Tpetra::Map<int,long> > map = Tpetra::createContigMap<int,long>(10,10,comm);
-  Array<long> lookup_gids(  tuple<long>(1,3,5) );
-  Array<int> expected_ids(  tuple<int>( 0,0,0) );
-  Array<int> nodeIDs( lookup_gids.size() );
+  RCP<const Tpetra::Map<LO,GO> > map = Tpetra::createContigMap<LO,GO>(10,10,comm);
+  Array<GO> lookup_gids(  tuple<GO>(1,3,5) );
+  Array<LO> expected_ids(  tuple<LO>( 0,0,0) );
```

Comment:
Line 129: expected_ids should be int, not LO, because these are MPI process ranks.